test/v1beta1/note_test.go

```diff
@@ -41,6 +46,162 @@ func TestNote(t *testing.T) {
 		})
 	})
 
+	t.Run("batch creating notes", func(t *testing.T) {
+		noteId1 := fake.UUID()
+		noteId2 := fake.UUID()
+
+		t.Run("should be successful", func(t *testing.T) {
+			batch, err := s.Gc.BatchCreateNotes(s.Ctx, &grafeas_go_proto.BatchCreateNotesRequest{
+				Parent: projectName,
+				Notes: map[string]*grafeas_go_proto.Note{
+					noteId1: createFakeBuildNote(),
+					noteId2: createFakeVulnerabilityNote(),
+				},
+			})
+			Expect(err).ToNot(HaveOccurred())
+
+			for _, o := range batch.Notes {
+				_, err = s.Gc.GetNote(s.Ctx, &grafeas_go_proto.GetNoteRequest{Name: o.GetName()})
+				Expect(err).ToNot(HaveOccurred())
+			}
+
+			t.Run("should not create notes with duplicate IDs", func(t *testing.T) {
```

Comment:
I don't think this should be nested under the successful test.

Reply:
I nested it because it's convenient that some notes have already been created, so this test can rely on that to ensure that they aren't duplicated, and that the non duplicate note was still created successfully.

Reply:
I just moved the setup (invoking `BatchCreateNotes`) into the parent and unnested.